content/en/docs/gke/deploy/deploy-cli.md

```diff
@@ -94,12 +94,13 @@ purpose. No tools will assume they actually exists in your terminal environment.
 
 1. Install [yq](https://github.com/mikefarah/yq)
 
+
    ```bash
    GO111MODULE=on go get github.com/mikefarah/yq/v3
    ```
 
    * If you don't have [Go](https://golang.org) installed you can download
-     a binary from [yq's GitHub releases](https://github.com/mikefarah/yq/releases).
+     a binary from [yq's GitHub releases](https://github.com/mikefarah/yq/releases), following the repository instructions. [Version 3.3.0](https://github.com/mikefarah/yq/releases/tag/3.3.0) is confirmed to work.
```

Comment:
The latest yq version appears to be v4.2.1 (https://github.com/mikefarah/yq/releases), so is the proposal here to add the min version (v3.3.0)?

Also, by "following the repository instructions", do you mean "follow the official installation steps" (https://github.com/mikefarah/yq#install)?

Thanks!

cc @Bobgy

Reply:
There were breaking changes in 4.x (`yq r` is no longer used). I would suspect that any 3.x version would work, but I have only tested it with 3.3.0.

Reply:
Yep with @sidpalas we both tested it with 3.3.0, since `yq r` has been removed in 4.x which was a breaking change for following the Kubeflow installation

Reply:
Yes, I confirmed the problem they met.
Shall we additional clarify do not use any 4.x versions? That seems to give more information than only 3.3.0 works.

Reply:
I've added some elaboration there, thank you!